azurelinuxagent/common/utils/restutil.py

```diff
@@ -170,6 +171,14 @@ def _get_http_proxy(secure=False):
     return host, port
 
 
+def cleanup_sas_tokens_from_urls(url):
+    redacted_text = "<SAS SIGNATURE>"
```

Comment:
Might be worth putting this in a constant somewhere, since it is referenced in the tests